src/python/test/test_dx_bash_helpers.py

```diff
@@ -269,41 +342,77 @@ def test_xattr_parameters(self):
 
             # Build the applet, patching in the bash helpers from the
             # local checkout
-            applet_id = build_app_with_bash_helpers(os.path.join(TEST_APPS, 'xattr_properties'), dxproj.get_id())
+            applet_id = build_app_with_bash_helpers(
+                os.path.join(TEST_APPS, "xattr_properties"), dxproj.get_id()
+            )
 
             # Run the applet
             applet_args = ["-iseq1=A.txt", "-iseq2=B.txt", "-iref=A.txt", "-iref=B.txt"]
-            cmd_args = ['dx', 'run', '--yes', '--watch', applet_id]
+            cmd_args = ["dx", "run", "--yes", "--watch", applet_id]
             cmd_args.extend(applet_args)
             run(cmd_args, env=env)
 
-    @unittest.skipUnless(testutil.TEST_RUN_JOBS, 'skipping test that would run a job')
+    def test_job_identity_token(self):
+        """Tests dx-jobutil-get-identity-token script"""
+        with temporary_project(
+            "TestDXBashHelpers.test_app1 temporary project"
+        ) as dxproj:
+            env = update_environ(DX_PROJECT_CONTEXT_ID=dxproj.get_id())
+
+            # Build the applet, patching in the bash helpers from the
+            # local checkout
+            applet_id = build_app_with_bash_helpers(
+                os.path.join(TEST_APPS, "job_identity_token"), dxproj.get_id()
+            )
```

Comment:
not sure if we have to use patched bash helpers here...perhaps just plain `dx build` would do the trick? (so it is not dependent on this directory)

Reply:
this is where I was told to place the test. Its an external bash script to dx toolkit so i think having a test in bash_helpers makes sense

Reply:
All of the standalone scripts are referred to as "bash helpers", even though this one is pure python. This helper ensures the remote applet installs `dxpy` to be the same as the checkout running the tests.